pkg/matcher/README.md

```diff
@@ -20,20 +33,31 @@ Short Syntax
      <expr>      ::= any string
 
  Long Syntax
-
-     <line>      ::= [ <not> ] <format> <separator> <expr>
+     [ <not> ] <format> <separator> <expr>
+     
      <format>    ::= [ 'string' | 'glob' | 'regexp' | 'simple_patterns' ]
      <not>       ::= '!'
                        negative expression
      <separator> ::= ':'
      <expr>      ::= any string
 ```
 
+**Tip**: In short syntax, for example, you can enable the glob format by starting the string with a `*`, while in the long syntax you need to define it more explicitly. The following examples are identical. `simple_patterns` can be used **only** with the long syntax.
+
+Short Syntax: `'* * '`
+Long Syntax: `'glob':'*'`
+
 ### String matcher
-The string matcher reports whether the given value equals to the string ( use == ).
+The string matcher reports whether the given value equals to the string ( use = ).
+
+Examples: 
+ - `'== foo'` will match only if the string is `'foo'`.
+ - `'[!] == bar'` will match any string that is not `'bar'`.
```

Comment:
it is one `=`

Reply:
Here we could add some tips.

String matcher means exact match. There are other string match related cases:

- has prefix
- has suffic
- contains

As you can guess it is achievable using `glob` matcher

- has prefix: `PREFIX*`
- has suffic: `*SUFFIX`
- contains: `*SUBSTRING*`

But it could be worth to mention in this section because these are kind of string matching cases.

What do you think?